capstone/src/main/webapp/js/servletConnection.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
you will probably have to add an `no-unused-vars` error ignore for some of these functions as they might be getting called from elsewhere. look at #100 or #35 for reference.